win/libsass.vcxproj

```diff
@@ -1,17 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="GitVersion;Main"
-    xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-
+<Project ToolsVersion="14.0" DefaultTargets="GitVersion;Main" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <SASSC_VERSION>[NA]</SASSC_VERSION>
     <LIBSASS_VERSION>[NA]</LIBSASS_VERSION>
   </PropertyGroup>
-
   <Target Name="GitVersion">
-
-    <Exec Command="git -C .. describe --abbrev=4 --dirty --always --tags > libsass-version" />
-    <Exec Command="git -C ../sassc describe --abbrev=4 --dirty --always --tags > sassc-version" />
-
+    <Exec Command="git -C .. describe --abbrev=4 --dirty --always --tags &gt; libsass-version" />
```

Comment:
Nah it's cool. It just stood out.
On 14 Aug 2015 02:19, "Adeel Mujahid" <notifications@github.com> wrote:

> In win/libsass.vcxproj
> <https://github.com/sass/libsass/pull/1439#discussion_r36993617>:
>
> >    <Target Name="GitVersion">
> > -
> > -    <Exec Command="git -C .. describe --abbrev=4 --dirty --always --tags > libsass-version" />
> > -    <Exec Command="git -C ../sassc describe --abbrev=4 --dirty --always --tags > sassc-version" />
> > -
> > +    <Exec Command="git -C .. describe --abbrev=4 --dirty --always --tags &gt; libsass-version" />
>
> @xzyfer <https://github.com/xzyfer>, I totally agree about the noise. It
> is just auto-formatting by VS, once we do that it won't get changed in
> future (just the end of file extra linefeed it omits in some cases, which
> we should keep for git). With that said, should I revert the extra
> changes?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sass/libsass/pull/1439/files#r36993617>.
>